db/migrate/1725874704270_add_quantity_size_color_to_cart_items.rb

```diff
@@ -0,0 +1,7 @@
+class AddQuantitySizeColorToCartItems < ActiveRecord::Migration[6.0]
+  def change
+    add_column :cart_items, :quantity, :integer, null: false
+    add_column :cart_items, :size, :string, null: false
+    add_column :cart_items, :color, :string, null: false
```

Comment:
Consider adding database indexes to 'size' and 'color' columns in the 'cart_items' table to improve query performance, especially if these fields will be used in search filters.